src/locales/zh.json

```diff
@@ -123,9 +123,11 @@
     "nervos_dao": {
       "nervos_dao": "NervosDAO",
       "total_deposit": "提现总额",
-      "transactions": "交易数量",
+      "deposit_transactions_count": "存款交易数量",
+      "withdraw_transactions_count": "提现交易数量",
       "depositor": "提现地址数量",
       "subsidy_granted": "补贴金额",
+      "total_depositors": "存款总额",
```

Comment:
depositors count not amount

Reply:
Updated. https://github.com/nervosnetwork/ckb-explorer-frontend/pull/360/commits/aa6df1932df50a82912ef1aed0b1880e477b12e7